<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color : rgb(202, 202, 202);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>268</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font: 75 20pt &quot;MS Shell Dlg 2&quot;;
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border - bottom: 2px solid black;
}

background-color :rgb(255, 0, 4);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>73</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;
background-color : rgb(121, 255, 130);
</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_min">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>143</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;
background-color : rgb(121, 255, 130);
</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_umn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>213</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;
background-color : rgb(121, 255, 130);
</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>73</y>
      <width>140</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;
background-color : rgb(121, 255, 130);
</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>213</y>
      <width>140</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;
background-color : rgb(121, 255, 130);
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ravn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>353</y>
      <width>140</width>
      <height>85</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;
background-color : rgb(121, 255, 130);
</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_str1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>143</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;
background-color : rgb(121, 255, 130);
</string>
    </property>
    <property name="text">
     <string>&lt;---</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_strv">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>73</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;
background-color : rgb(121, 255, 130);
</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_podr">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>213</y>
      <width>70</width>
      <height>225</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;
background-color : rgb(121, 255, 130);
</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_del">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>283</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;
background-color : rgb(121, 255, 130);
</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_del2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>353</y>
      <width>70</width>
      <height>85</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;
background-color : rgb(121, 255, 130);
</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
